sdk/python/tests/compiler/testdata/withitem_nested.yaml

```diff
@@ -0,0 +1,188 @@
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: my-in-coop1
+spec:
+  params:
+  - name: loop-item-param-00000001-subvar-a
```

Comment:
do we really need 8 digits?

Reply:
It was defined in LoopArguments of kfp.dsl._for_loop.py, not by us:
```
class LoopArguments(dsl.PipelineParam):
    ... 
    # number of characters in the code which is passed to the constructor
    NUM_CODE_CHARS = 8

```

Reply:
ah, ok, thanks!